test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
+		} );
+
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
+		} );
+
+		test.afterEach( async ( { requestUtils } ) => {
+			await requestUtils.deleteAllPosts();
+		} );
+
+		test( 'default to a list of pages if there are no menus', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.insertBlock( { name: 'core/navigation' } );
+
+			// Check Page List is in the list view.
+
+			// Open the list view.
+			await page.click( '[aria-label="Document Overview"i]' );
+
+			// Click the Navigation block expander, we need to use force because it has aria-hidden set to true.
+			await page.click(
+				`//a[.//span[text()='Navigation']]/span[contains(@class, 'block-editor-list-view__expander')]`,
+				{ force: true }
+			);
+
+			// Find the Page list selector inside the navigation block.
+			const pageListSelector = await page.locator(
+				`//table[contains(@aria-label,'Block navigation structure')]//a[.//span[text()='Page List']]`
+			);
+
+			expect( pageListSelector ).toBeTruthy();
```

Comment:
I believe this will provide a false positive in the sense that the `locator` will always return an object which is therefore truthy.

What I think we want is `await expect( pageListSelector ).toBeVisible();` but this will currently fail the test becayse the request (the Nav block loading state) won't have resolved.

Reply:
Fixed.